mastering/revisions.rst

```diff
@@ -0,0 +1,93 @@
+Package Revisions [EXPERIMENTAL]
+================================
+
+The goal of the `revisions feature` is to achieve package immutability, so nothing in a server is ever overwritten.
+
+.. note::
+
+    This is the first piece to achieve reproducibility: Recreate the exact dependency graph by using some
+    mechanism like a ``graph lock`` file. For example, if we store a ``graph lock`` file for the different releases
+    of our project, we can install the same dependencies just by using the graph lock.
+
+    **IMPORTANT:** The reproducibility is in the Conan roadmap and currently under development.
```

Comment:
Please include the standard experimental warning like other features

Reply:
Example please

Reply:
Note that the IMPORTANT is not about an experimental anything, but a roadmap hint

Reply:
```
.. warning::

    This is an **experimental** feature subject to breaking changes in future releases.
```